src/test/java/org/fundacionjala/pivotal/hooks/Hooks.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.pivotal.hooks;
+
+import cucumber.api.java.After;
+import org.fundacionjala.pivotal.api.APIProjectsRequests;
+import static org.fundacionjala.pivotal.steps.api.APIProjectSteps.PROJECT_ENDPOINT;
+
+/**
+ * Hooks.java
+ * Class that will have hooks for scenarios.
+ */
+public class Hooks {
+    private APIProjectsRequests requests;
+
+    /**
+     * After hook to delete the project created in tests.
+     */
+    @After("@PostConditionProject")
+    public void afterCreatingProject() {
+        requests = new APIProjectsRequests();
+        requests.setBaseUri("https://www.pivotaltracker.com/services/v5");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)